assets/js/util.js

```diff
@@ -120,7 +120,10 @@ $(document).ready(function () {
     resizeFooter();
     $(window)
         .on('hashchange', persistChoices)
-        .resize(resizeFooter);
+        .resize(function () {
+            resizeFooter();
+            populateTranslationList();
```

Comment:
Yep! debouncing will ensure that `populateTranslationList` gets called only after say N seconds pass after the function is invoked! This could significantly improve the performance. On it.